libs/ledgerjs/packages/react-native-hw-transport-ble/src/remapErrors.ts

```diff
@@ -11,8 +17,33 @@ export const remapError = (error: Error | null | undefined) => {
 
   return error;
 };
-export const rethrowError = (e: Error | null | undefined) => {
+
+export const rethrowError = (e: Error | null | undefined): never => {
   throw remapError(e);
 };
+
 export const decoratePromiseErrors = <A>(promise: Promise<A>): Promise<A> =>
   promise.catch(rethrowError);
+
+export const bleErrorToHwTransportError = new Map([
+  [BleErrorCode.ScanStartFailed, HwTransportErrorType.BleScanStartFailed],
+  [
+    BleErrorCode.LocationServicesDisabled,
+    HwTransportErrorType.BleLocationServicesDisabled,
+  ],
+  [
+    BleErrorCode.BluetoothUnauthorized,
+    HwTransportErrorType.BleBluetoothUnauthorized,
+  ],
+]);
```

Comment:
FYI: i updated a bit `HwTransportErrorType` and which error type maps to which errors [here](https://github.com/LedgerHQ/ledger-live/pull/2912/commits/4e18e876d278b854da1b1de6b35554dd98a6f084#diff-a53dc71468c99439950e35dcf055a811bf227ce35eb8a9d3d9cfdcb227200fcb) <- depending which PR is merged first, i'll help on this ;)